### PR TITLE
Add 19.41 support

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -47,12 +59,16 @@ cl_version:
...
  - 1754ea58-11a6-44ab-a262-696e194ce543
  - 1754ea58-11a6-44ab-a262-696e194ce543
  - 71c6392f-8df5-4b61-8d50-dba6a525fb9d
  - c7707d68-d6ce-4479-973e-e2a3dc4341fe
  - b929b7fe-5c89-4553-9abe-6324631dcc3a
 sha256:
+ - 15B8F5B2106DC7A7BD83AB57B796770E0F4ECB891AD19BF655C9D6A9DA650AD2
+ - 3642E3F95D50CC193E4B5A0B0FFBF7FE2C08801517758B4C8AEB7105A091208A
  - 15B8F5B2106DC7A7BD83AB57B796770E0F4ECB891AD19BF655C9D6A9DA650AD2
  - 3642E3F95D50CC193E4B5A0B0FFBF7FE2C08801517758B4C8AEB7105A091208A
```
Comment (h-vetinari) [inline recipe/conda_build_config.yaml:73]: I had begun preparing a PR for this a while ago, but it seemed to me that they haven't updated the `vc_redist` link yet? Pointing to the exact same uuic/sha sounds suspicious, no?

Comment (isuruf) [reply recipe/conda_build_config.yaml:73]: > Pointing to the exact same uuic/sha sounds suspicious, no?

Not really. Some compiler updates don't come with runtime updates.

Comment (h-vetinari) [reply recipe/conda_build_config.yaml:73]: OK, fair enough; can't remember such a case in recent memory though. 🤷 

Separately, the GHA/AZP images are [still](https://github.com/actions/runner-images/blob/d33c9005ccc308dfce8ff1710cc1fcbc5c6b22ac/images/windows/Windows2022-Readme.md#visual-studio-enterprise-2022) on 17.10/19.40 AFAICT

Comment (isuruf) [reply recipe/conda_build_config.yaml:73]: They briefly updated it (See https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/1010969/logs/24) and reverted it

**Comment by isuruf** [comment]
Mind sending a PR to mark them unbroken?

**Comment by jdblischak** [comment]
Sure. I'll do that now

**Comment by jdblischak** [comment]
This is chaos. Azure keeps switching the image version. Last night at 9:30 pm ET it was using 20240825.1.0 (Visual Studio 17.11), but then exactly 12 hours later this morning at 9:30 am ET, it reverting to using image version 20240811.1.0 (Visual Studio 17.10).

My questions are: 

* Should we report this upstream? It's hard to imagine that only conda-forge jobs are being disrupted by the random switching between Visual Studio 17.10 and 17.11. I couldn't find any open Issue. The closest thing I could find was this Issue requesting Visual Studio 17.11 https://github.com/actions/runner-images/issues/10448

* Is there any way to make this recipe (or conda itself) aware of the version(s) of Visual Studio available on the machine? These errors happen every time Azure updates the version of Visual Studio in its image. Having to exactly align our vc-feedstock release with their image update is difficult, to say the least. I assume it won't be possible, but I wanted to at least raise the idea

xref: https://github.com/TileDB-Inc/conda-forge-nightly-controller/issues/133#issuecomment-2315592291

**Comment by jdblischak** [comment]
Unfortunately it looks like we can't pin the exact image version for the Azure build, so that isn't an option as a short-term workaround https://github.com/actions/runner-images/discussions/3464

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 {% if vsyear is not defined %}
 {% set vsyear = "" %}
-{% set runtime_version = "" %}
+{% set runtime_version = "14.40.33810" %}
```
Comment (mgovers) [inline recipe/meta.yaml:12]: i think this broke the 
```yaml
cl_version:
 - 19.41.34120
```
which may have caused #83

**Pull request merged**